extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -192,18 +192,26 @@ amp-story-page.i-amphtml-story-page-shown {
   transition: 300ms transform linear!important;
 }
 
-.i-amphtml-story-button-move:hover {
-  opacity: 1!important;
+.i-amphtml-story-button-container.prev-container:hover ~ amp-story-page[pre-active] {
```

Comment:
Is there a way to avoid using a tilde selector? Worried about maintainability and possibly performance.

Reply:
It will probably involve jS although I wanted to keep it to css itself.

WDYT?

Reply:
Let's keep as is for now.